src/templates/Dataset/dataset.test.jsx

```diff
@@ -53,7 +55,21 @@ describe('<Dataset />', () => {
     });
     await waitFor(() => {
       expect(screen.getByRole('heading', { name: '2022 General Payment Data' }));
-      expect(screen.queryAllByText('Data Dictionary'));
+      expect(screen.getByTestId('dataset-dictionary-tab')).toBeInTheDocument();
     });
-  })
+  });
+  test("does not render data dictionary", async () => {
```

Comment:
can this be a little more descriptive - something like "Does not render Data Dictionary if dataset does not have describedBy attribute"

Reply:
addressed

https://github.com/GetDKAN/cmsds-open-data-components/pull/241/commits/6a865e8861b5557b66dcbed2b6b4cf622e216e04